docs/authentication/session-management-overview.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Session Management overview
+id: session-management-overview
+description: ''
+slug: /session-management-overview
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/08/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Session Management
+
+Session management is a crucial aspect of web application development. An authentication session is a period of user interaction with the application, typically starting with the user providing credentials and ending with logout or inactivity timeout. Effective session management ensures user data privacy, maintains session state, and provides a seamless user experience.
+
+The Beyond Identity session consists of something we call a `token`, specifically [access token](/docs/next/api-token-overview)s. An access token, also called an API token, is an object or file that enables access to an application or resource. The token is a JWT, which is digitally signed and contains information about what resources can be accessed and for how long.
+
+When you request an [access token](https://datatracker.ietf.org/doc/html/rfc6749#section-1.4), it is possible to receive an additional token called a [refresh token](https://datatracker.ietf.org/doc/html/rfc6749#section-1.5) in the response. The refresh token can be used to request additional access tokens with identical or narrower scope.
+
+## How to add session management to my application
+
+There are a few functions of tokens that you should be aware of.
+
+- [Create a Token](/docs/next/create-api-token) (Pre-Requisite)
+- [Store Tokens](#how-to-store-tokens)
```

Comment:
Perhaps add "Create cookie" or "Create session cookie" with a link to that section below

Reply:
Added the link. Confirming, does this go to "Where to store tokens"?

Reply:
I suggest that it link to a section specific to cookies such as Session Cookie Strategy, though i see that you mention storing tokens in cookies within the Where to store tokens section. I'm a little confused right now as to whether we are recommending that the reader store access and refresh tokens "in HTTP-only cookies" or store them server side with a session id in the cookie, or a combination. Sorry maybe it's me. Can you clarify the role of cookies relative to the tokens?

Reply:
I think in either case we are recommending that the reader use session cookies, but they can use them either to
- Store the tokens on the client
- Store an identifier on the client, which represents the tokens that are stored on the server

I have probably been referencing too many documents and resources on the subject. @tomsik68, do you know if we have a Beyond Identity recommended approach?

I wonder if it would make more sense to swap the Session Cookie Strategy and the Where to Store Tokens sections.

Reply:
> I think in either case we are recommending that the reader use session cookies

I agree with this. There's not too many options in terms of how to persist the session on browser side. 

> do you know if we have a Beyond Identity recommended approach?

As far as I'm aware, there's no BI-recommended approach on how to store tokens.

Reply:
I was thinking about this, and I think the distinction really comes down to whether the application wants to handle session management on the client or the server. If on the client, then session cookies store tokens. If on the server, then session cookies store ids. Does that make more sense? Any recommendations on how to convey that better?